### PR TITLE
technical preview release demo pipeline

### PR DESCRIPTION
**Pull request by bwsw**

### CODE REVIEW

File: samples/peoplenet_detector/overlay.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+class Overlay(NvDsDrawFunc):
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.font_face = cv2.FONT_HERSHEY_SIMPLEX
+        self.font_scale = get_font_scale(
+            self.counters_height, self.counters_font_thickness, self.font_face
+        )
+
+        self.logo = load_sprite(
```
Comment (denisvmedyantsev) [inline samples/peoplenet_detector/overlay.py:18]: The logo is not a sprite, I think better to add separate function and do not confuse users.

File: savant/utils/artist/artist_gpumat.py [hunk 2]
```diff
@@ -196,84 +200,90 @@ def add_polygon(
...
+            cv2.drawContours(
+                self.overlay, [vertices], 0, convert_color(line_color), line_width
+            )
 
-    def blur(self, bbox: BBox, padding: int = 0):
+    def blur(self, bbox: BBox, padding: int = 0, sigma: Optional[float] = None):
         """Apply gaussian blur to the specified ROI.
 
         :param bbox: ROI specified as Savant bbox.
         :param padding: Increase the size of the rectangle in each direction,
```
Comment (denisvmedyantsev) [inline savant/utils/artist/artist_gpumat.py:212]: sigma param description?

File: savant/utils/artist/position.py [hunk 3]
```diff
@@ -18,7 +18,12 @@ class Position(Enum):
 
 
 def get_text_origin(
-    anchor_point: Position, anchor_x: int, anchor_y: int, text_w: int, text_h: int
+    anchor_point: Position,
+    anchor_x: int,
+    anchor_y: int,
+    text_w: int,
+    text_h: int,
+    baseline: int,
```
Comment (denisvmedyantsev) [inline savant/utils/artist/position.py:26]: baseline param description?

**Review by denisvmedyantsev — CHANGES_REQUESTED** [review]

**Comment by denisvmedyantsev** [comment]
What is the reason for smpte100_1280x900.jpeg in sample module dir? May be better to add this image as the default stub for the adapter? There is currently no code to run the demo and adapters, so this image looks redundant now.

File: savant/utils/artist/position.py (hunk 3 above)
Comment (abramov-oleg) [reply savant/utils/artist/position.py:26]: Added.

File: savant/utils/artist/artist_gpumat.py (hunk 2 above)
Comment (abramov-oleg) [reply savant/utils/artist/artist_gpumat.py:212]: Added the description.

File: samples/peoplenet_detector/demo.yml [hunk 4]
```diff
@@ -0,0 +1,80 @@
+name: ${oc.env:MODULE_NAME, 'demo'}
```
[the remark this answers is not in the document]
Comment (abramov-oleg) [reply samples/peoplenet_detector/demo.yml:2]: Added comments for module config values.

File: samples/peoplenet_detector/config_tracker_NvSORT.yml [hunk 5]
```diff
@@ -0,0 +1,65 @@
+%YAML:1.0
```
[the remark this answers is not in the document]
Comment (abramov-oleg) [reply samples/peoplenet_detector/config_tracker_NvSORT.yml:1]: Added a note.

File: samples/peoplenet_detector/analytics.py [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+        """On source EOS event callback."""
+        del self.person_counters[(source_id, 'face')]
+        del self.person_counters[(source_id, 'noface')]
+
+    def process_frame(self, buffer: Gst.Buffer, frame_meta: NvDsFrameMeta):
+        person_bboxes = []
+        face_bboxes = []
+        for obj_meta in frame_meta.objects:
+            if obj_meta.is_primary:
+                continue
```
[the remark this answers is not in the document]
Comment (abramov-oleg) [reply samples/peoplenet_detector/analytics.py]: Refactored to save the reference in a var, removed 2nd loop.

File: samples/peoplenet_detector/analytics.py [hunk 7]
```diff
@@ -0,0 +1,54 @@
+from collections import defaultdict
```
[the remark this answers is not in the document]
Comment (abramov-oleg) [reply samples/peoplenet_detector/analytics.py:2]: Added docstrings, reformatted.

File: samples/peoplenet_detector/overlay.py (hunk 1 above)
Comment (abramov-oleg) [reply samples/peoplenet_detector/overlay.py:39]: Sprite is just a piece of 2d graphics, both logo image and animation frames can be considered sprites.

I think it's fine as it is.

**Review by denisvmedyantsev — APPROVED** [review]

**Pull request merged**